handsontable/src/core.js

```diff
@@ -4371,6 +4385,166 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     return instance.isLtr() ? 1 : -1;
   };
 
+  /**
+   * Returns the ShortcutManager's instance.
+   *
+   * @memberof Core#
+   * @function getShortcutManager
+   * @returns {object}
+   */
+  this.getShortcutManager = function() {
+    return shortcutManager;
+  };
+
+  const gridContext = shortcutManager.addContext('grid');
+  const gridConfig = {
+    runAction: () => isDefined(instance.getSelected()),
+    namespace: SHORTCUTS_NAMESPACE,
+  };
+
+  shortcutManager.setActiveContextName('grid');
+
+  gridContext.addShortcuts([{
+    variants: [['Control', 'A'], ['Meta', 'A']],
+    callback: (event) => {
+      instance.selectAll();
+      event.preventDefault();
+    },
+    preventDefault: false,
```

Comment:
I think that it can be simplified to this
```suggestion
    callback: () => {
      instance.selectAll();
    },
```

Reply:
Good catch, it was code introduced before adding an extra `editor` context. Changed within: https://github.com/handsontable/handsontable/commit/7f09e82adcf0ef9c60482b1f2f57d3b0bd89e412.